internal/pkg/mount/v2/pseudo.go

```diff
@@ -21,6 +21,9 @@ func Pseudo() Points {
 		NewPoint("tmpfs", "/run", "tmpfs", WithFlags(unix.MS_NOSUID|unix.MS_NOEXEC|unix.MS_RELATIME), WithData("mode=0755")),
 		NewPoint("tmpfs", "/system", "tmpfs", WithData("mode=0755")),
 		NewPoint("tmpfs", "/tmp", "tmpfs", WithFlags(unix.MS_NOSUID|unix.MS_NOEXEC|unix.MS_NODEV), WithData("size=64M"), WithData("mode=0755")),
+		// TODO: mount this under /sys/fs/selinux as in most distros?
+		// That'd be harder to move when we switch root
+		NewPoint("selinuxfs", "/selinux", "selinuxfs", WithFlags(unix.MS_NOSUID|unix.MS_NOEXEC|unix.MS_RELATIME)),
```

Comment:
yes, let's use standard paths as tools like cilium/csi's might try to mount this whem selinux is enabled to gather info